src/components/SideNavigation/NavigationItem/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import classNames from 'classnames'
+import { NavigationItemTypes, TNavigationLinkPropTypes } from './types'
+import './index.scss'
+import Icon from '../../Icon'
+import Badge from '../../Badge'
+
+const NavigationItem = (props: TNavigationLinkPropTypes) => {
+  const {
+    As,
+    type,
+    open,
```

Comment:
open is like function name, suggest to rename `isNavigationOpen`

Reply:
If you agree I suggest use isOpen prop instead of isNavigationOpen because we inside of Navigation component and it will it is clear who it refers to

Reply:
done!